src/utils/filters.ts

```diff
@@ -0,0 +1,154 @@
+// @ts-ignore
+import { geojsonToWKT } from "@terraformer/wkt";
+import { Geometry } from "geojson";
+
+export {
+  and,
+  or,
+  intersects,
+  not,
+  equals,
+  lessThan,
+  lessThanOrEqual,
+  greaterThan,
+  greaterThanOrEqual,
+  like,
+  between,
+};
```

Comment:
I'm torn about whether we should be abstracting the CQL filters behind functions like this. It made sense when the filters were XMLs and really tricky to handle, but CQL seems to be very human readable and might be easier to handle directly rather than using these functions. The only exception is for the geospatial intersects filter - it's good to abstract the wkt conversion.

Reply:
resolving (just because its a duplicate)